src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -31,14 +33,18 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
-        shoppingCartService.addSession(
-                movieSessionService.get(movieSessionId), userService.get(userId));
+    public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {
```

Comment:
```suggestion
    public void addToCart(Authentication authentication, @RequestParam @Min(1) Long movieSessionId) {
```

Reply:
add these in every method that requires some kind of id

Reply:
done